coffee-cdi/src/main/java/hu/icellmobilsoft/coffee/cdi/logger/LogContainer.java

```diff
@@ -367,4 +369,17 @@ public enum LogLevel {
          */
         ERROR,
     }
+
+    private void cleanOlderLogs() {
+        if (logs.size() >= LOG_MAX_SIZE) {
+            for (int i = 1; i <= logs.size() - LOG_MAX_SIZE; i++) {
+                logs.remove(0);
```

Comment:
for ciklusban egyesével a lista elejéről törölgetés nagyon nem hatékony, jobb azt egyben letudni:
`logs.subList(0, countToRemove).clear();`